### ISSUE TITLE
Invalid JSON (Synapse)

### ISSUE THREAD

**Issue by jwh** [hifi/heisenbridge#174]
Hi,

Recently my bridge has stopped working - I *think* this coincides with the update to synapse 1.47.0, which manifests like this:

```
Traceback (most recent call last):
  File "/root/heisenbridge/lib/python3.9/site-packages/heisenbridge/__main__.py", line 171, in _on_mx_event
    await room.on_mx_event(event)
  File "/root/heisenbridge/lib/python3.9/site-packages/heisenbridge/room.py", line 106, in on_mx_event
    await handler(event)
  File "/root/heisenbridge/lib/python3.9/site-packages/heisenbridge/private_room.py", line 628, in on_mx_message
    await self.serv.api.post_room_receipt(event["room_id"], event["event_id"])
  File "/root/heisenbridge/lib/python3.9/site-packages/heisenbridge/matrix.py", line 254, in post_room_receipt
    return await self.call("POST", f"/_matrix/client/r0/rooms/{room_id}/receipt/{receipt_type}/{event_id}")
  File "/root/heisenbridge/lib/python3.9/site-packages/heisenbridge/matrix.py", line 92, in call
    raise self._matrix_error(ret)
heisenbridge.matrix.MatrixError: M_NOT_JSON
```
and on synapse:

```
2021-11-22 16:36:09,025 - synapse.http.servlet - 620 - WARNING - POST-69100 - Unable to parse JSON: Expecting value: line 1 column 1 (char 0) (b'')
```

I'm not sure if I can dump the json easily, unless there is a heisenbridge flag I can pass to have it spew out debug?

While control rooms work fine, any attempt to send (or receive) messages in joined irc channels/PMs cause the above

**Comment by hifi** [comment]
That could be a Synapse update as well, I don't think heisen has ever posted an actual body with the read receipt call.

**Comment by hifi** [comment]
Right, I can probably fix that in a few hours and test with synapse 1.47.0. Fun sudden change...

**Comment by jwh** [comment]
Awesome, if you want me to test let me know (@jwh:covert.tel on matrix)

**Comment by hifi** [comment]
And this was fixed in v1.7.1. Thanks.
